[TASK] Setup note added in readme (#23)
pluswerk/mail_logger: Readme.md

2. [E-mail templates](#2-e-mail-templates)
3. [E-mail debugging](#3-e-mail-debugging)

## Extension installation

> Just copy files to TYPO3 Extension folder and activate it in backend.

Reviewer: ```suggestion
Just copy the files into the TYPO3 extension folder and activate it in the backend.
```

> Just copy the files into the TYPO3 extension folder and activate it in the backend.

Reviewer 2: can we explain the Installation via composer. As it is the preferred installation method.

Author: Done

> Then add the TypoScript files in backend.

Reviewer: ```suggestion
Then add the TypoScript files to the backend.
```
This sentence might make sense to you but most-likely confuses others

> Now everything is ready and you can create your own mailing settings as TypoScript.

Reviewer: ```suggestion
Now everything is set-up and ready for you to create your own mailing settings in TypoScript.
```
